sdk/lib/drivers/ip/transport/tcp/tcp.c

```diff
@@ -379,10 +379,18 @@ NTSTATUS TCPConnect
         }
 
         /* Allocate the port in the port bitmap */
-        Connection->AddressFile->Port = TCPAllocatePort(LocalAddress.Address[0].Address[0].sin_port);
+        UINT nPort = TCPAllocatePort(LocalAddress.Address[0].Address[0].sin_port);
 
-        /* This should never fail */
-        ASSERT(Connection->AddressFile->Port != 0xFFFF);
+        if (nPort == (UINT)-1)
```

Comment:
Same as above here, why is this suddenly OK now?

Reply:
Same answer as above.